baremaps-openapi/src/main/resources/redoc.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>ReDoc</title>
+  <!-- needed for adaptive design -->
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+<redoc spec-url='http://localhost:9000/swagger.json'></redoc>
```

Comment:
Following the OGC API this should point to `http://localhost:9000/api`

Reply:
Ok, this is technically not true. On the other hand `swagger.json` is equally wrong. Finally we should not forget to link these resources on the landing page.